Packs/ShiftLeft/Integrations/shiftleft/shiftleft.py

```diff
@@ -0,0 +1,338 @@
+
+    def get_scopes(self, org_id: str):
+        """Get user scopes"""
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/orgs/{org_id}/my_scopes",
+        )
+
+    def list_apps(
+        self,
+        org_id: str,
+    ) -> Dict[str, str]:
+        """Returns list of apps"""
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/orgs/{org_id}/apps",
+        )
+
+    def list_app_findings(
+        self,
+        org_id: str,
+        app_name: str,
+        severity: Union[str, List[str], None],
+        type: Union[str, List[str], None],
+        version: str,
+    ) -> Dict[str, str]:
+        """Returns list of findings"""
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/orgs/{org_id}/apps/{app_name}/findings",
+            params={"severity": severity, "type": type, "version": version},
+        )
+
+
+def test_module(client: ShiftLeftClient, org_id: str) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ""
+    try:
+        result = client.get_scopes(org_id)
+        message = "ok" if result and result.get("ok") else "Unauthorized"
+    except DemistoException as e:
+        if "Forbidden" in str(e) or "Authorization" in str(e):
+            message = "Authorization Error: make sure access token is correctly set"
+        else:
+            raise e
+    return message
+
+
+def list_apps_command(
+    client: ShiftLeftClient, org_id: str, args: Dict[str, Any]
+) -> CommandResults:
+    result = client.list_apps(org_id)
+    apps: Any = result.get("response") if result.get("ok") else []
+    for a in apps:
+        if a.get("tags"):
+            a["labels"] = "\n".join(
+                [f'`{t.get("key")}`: {t.get("value")}' for t in a.get("tags")]
+            )
+    if apps:
+        markdown = f"### Apps List ({len(apps)})\n"
+        markdown += tableToMarkdown(
+            "",
+            apps,
+            headers=[
+                "id",
+                "name",
+                "labels",
+            ],
+        )
+        return CommandResults(
+            readable_output=markdown,
+            outputs_prefix="ShiftLeft.Apps",
+            outputs_key_field="id",
+            outputs=apps,
+        )
+    else:
+        return CommandResults(
+            outputs_prefix="ShiftLeft.Apps",
+            outputs_key_field="",
+            outputs={},
+        )
+
+
+def list_app_secrets_command(
+    client: ShiftLeftClient, org_id: str, args: Dict[str, Any]
+) -> CommandResults:
+    app_name = args.get("app_name", None)
```

Comment:
If the value does not exist, the get will set None as default.
```suggestion
    app_name = args.get("app_name")
```